draft-ietf-moq-transport.md

```diff
@@ -813,7 +813,7 @@ formatted as follows:
 ~~~
 MOQT Control Message {
   Message Type (i),
-  Message Payload (..),
+  Message Payload (b),
```

Comment:
This change means any Message Type with no payload will still have a trailing length of 0. Maybe make the payload [optional] to avoid that, although I don't really expect a future message with no payload.

Reply:
yeah.. that's my thinking too . So will keep this as-is